### ISSUE TITLE
Understanding 1.4.11 - recent additional image incorrect

### ISSUE THREAD

**Issue by patrickhlauke** [w3c/wcag#2061]
Recently https://github.com/w3c/wcag/pull/2028 this new example has been added to the 1.4.11 understanding https://w3c.github.io/wcag/understanding/non-text-contrast.html

> Note that this Success Criterion does not directly compare the focused and unfocused states of a control - if the focus state relies on a change of color (e.g., changing only the background color of a button), this Success Criterion does not define any requirement for the difference in contrast between the two states.
>
> ![Three blue buttons, the center button is a lighter blue than the others.](https://user-images.githubusercontent.com/895831/135358653-e3875a21-df73-4d8b-87cf-f95b80a13b1e.png)
>
> Figure 15 The change of background within the component is not in scope of non-text contrast. However, this would not pass Use of color.

However, the image doesn't match up with the prose - as it's not *just* the background that changes here, but the colour of the text as well (and a dramatic change here from white to black, the most contrasty change possible).

Suggest changing the image so that the three buttons are the same *except for* a change of *just* the background

**Comment by filippo-zorzi** [comment]
@patrickhlauke I think this issue should be closed but I don't know which PR closed it.

**Comment by melaniephilipp** [comment]
For me, this is fixed in the 2.2 version: https://www.w3.org/WAI/WCAG22/Understanding/non-text-contrast but not in the 2.1 version https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast

**Comment by filippo-zorzi** [comment]
You're right! I didn't check the 2.1

**Comment by kfranqueiro** [comment]
WCAG 2.1 informative docs have now been republished with this fix, made possible by #4007.
